src/components/table/TableMobileSort.vue

```diff
@@ -28,6 +28,7 @@
                         v-show="currentSortColumn === mobileSort"
                         :class="{ 'is-desc': !isAsc }"
                         icon="arrow-up"
+                        :pack="iconPack"
```

Comment:
There is `both` prop, you doesn't need to set icon pack.

Reply:
Ok fair enough. But using FontAwesome 5 SVG does not seem to work with this specific component (the `Table` component in Desktop view works perfectly with the `arrow-up` and `arrow-down` icons when specifying `:icon-pack="fas"`). Any ideas what's the issue then?

Reply:
Did you set fas in the constructor options?

Reply:
Ok now I feel silly. Using `nuxt-buefy`, I had only `defaultIconComponent: 'font-awesome-icon'` in the constructor options, working almost everywhere. Only adding `defaultIconPack: 'fas'` fixed the issue indeed, thanks!

Sorry for the noise, I'll close this PR.